react/List.tsx

```diff
@@ -98,97 +98,100 @@ const StoreList = ({
       center,
     })
   }
 
-  if (called) {
-    if (!loading && !!data && data.getStores.items.length === 0) {
-      state.strikes < 4 &&
-        setState((prev) => ({
-          ...prev,
-          strikes: ++prev.strikes,
-        }))
-    }
-
-    if (!state.center && data?.getStores?.items.length) {
-      const [firstResult] = data.getStores.items
+  if (!state.center && data?.getStores?.items.length) {
+    const [firstResult] = data.getStores.items
 
-      const { latitude, longitude } = firstResult.address.location
+    const { latitude, longitude } = firstResult.address.location
 
-      const center = ofData?.shippingData?.address?.geoCoordinates ?? [
-        longitude || long,
-        latitude || lat,
-      ]
+    const center = ofData?.shippingData?.address?.geoCoordinates ?? [
+      longitude || long,
+      latitude || lat,
+    ]
 
-      handleCenter(center)
-    }
+    handleCenter(center)
+  }
 
-    const stores =
-      data?.getStores?.items.sort((a, b) => {
-        if (a[sortBy] < b[sortBy]) {
-          return -1
-        }
-
-        if (a[sortBy] > b[sortBy]) {
-          return 1
-        }
-
-        return 0
-      }) ?? []
-
-    return (
-      <div className={`flex flex-row ${handles.container}`}>
-        <div className={`flex-col w-30 ${handles.storesListCol}`}>
-          {loading && <Spinner />}
-          {!loading && !!data && stores.length > 0 && (
-            <div className={`overflow-auto h-100 ${handles.storesList}`}>
-              <Listing items={stores} onChangeCenter={handleCenter} />
-              {state.allLoaded && (
-                <span
-                  className={`mt2 link c-link underline-hover pointer ${handles.loadAll}`}
-                  onClick={() => {
-                    loadAll()
-                  }}
-                >
-                  <FormattedMessage id="store/load-all" />
-                </span>
-              )}
-            </div>
-          )}
-          {!loading && !!data && stores.length === 0 && (
-            <div className={handles.noResults}>
-              <h3>
-                <FormattedMessage id="store/none-stores" />
-              </h3>
-            </div>
-          )}
-        </div>
-        <div className={`flex-col w-70 ${handles.storesMapCol}`}>
-          {!loading &&
-            !!data &&
-            stores.length > 0 &&
-            googleMapsKeys?.logistics?.googleMapsKey && (
-              <Pinpoints
-                apiKey={googleMapsKeys.logistics.googleMapsKey}
-                className={handles.listingMapContainer}
-                items={data.getStores.items}
-                zoom={state.zoom}
-                center={state.center}
-                icon={icon}
-                iconWidth={iconWidth}
-                iconHeight={iconHeight}
-              />
+  const stores =
+    data?.getStores?.items.sort((a, b) => {
+      if (a[sortBy] < b[sortBy]) {
+        return -1
+      }
+
+      if (a[sortBy] > b[sortBy]) {
```

Comment:
We can make this a bit easier to read with if.. else if... else

Reply:
ESLint doesn't like that 🤷